src/main/java/uk/gov/ons/ctp/response/action/endpoint/ActionRuleEndpoint.java

```diff
@@ -89,8 +89,7 @@
       final @RequestBody @Valid ActionRulePostRequestDTO actionRulePostRequestDTO,
       final BindingResult bindingResult)
       throws CTPException, InvalidRequestException {
-    log.debug(
-        "Entering createActionRule with actionRulePostRequestDTO {}", actionRulePostRequestDTO);
+    log.debug("Entering createActionRule");
```

Comment:
we're logging less info than we were before

Reply:
actionRulePostRequestDTO added